sdk/python/feast/permissions/client/auth_client_manager.py

```diff
@@ -39,7 +39,7 @@ def get_auth_client_manager(auth_config: AuthConfig) -> AuthenticationClientMana
         )
 
 
-def create_metadata(auth_config: AuthConfig) -> Tuple[Tuple[str, str]]:
+def create_grpc_auth_header(auth_config: AuthConfig) -> Tuple[Tuple[str, str]]:
```

Comment:
maybe name just `create_auth_header`

Reply:
In general I agree I am just not sure the format in which it is returned Tuple[Tuple[str, str]] is compatible with other protocols